lib/src/repository_pocketbase.dart

```diff
@@ -44,19 +77,18 @@ class RepositoryPocketBase<T> extends Repository<T> {
 
     try {
       final data = toPocketBase(item.object);
+      data['id'] = item.id;
```

Comment:
we should not set any fields, we have now hardcoded a field of id. The use must use the autoidentify methods with the updater functions if they want the fields set, here we must do exactly what they told us, add the object given.

Reply:
fixed